styles/global.css

```diff
@@ -27,14 +28,15 @@ body {
   background-color: black;
   position: fixed;
   height: 60px;
+  z-index: 1000;
 }
 .logo {
   display: flex;
   flex-direction: column;
   justify-content: center;
   align-items: center;
   height: 100%;
-  margin: 20px;
+  margin: 30px;
```

Comment:
Do you mean margin-left here? 
Right and bottom won't have any effect

Reply:
Yes, only left side needs margin